blob_writer.js

```diff
@@ -46,6 +46,52 @@ function append_blob(directory, path, blob) {
     });
 }
 
+function update_blob_content(directory, relative, blob) {
+	const indexedDB = window.indexedDB ||
+        window.mozIndexedDB ||
+        window.webkitIndexedDB ||
+        window.msIndexedDB;
+
+	return new Promise((resolve, reject) => {
+		const connection = indexedDB.open("Disc");
```

Comment:
Should we include a workaround for this catastrophic iOS 14.1 bug?
https://bugs.webkit.org/show_bug.cgi?id=226547